<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mw</class>
 <widget class="QMainWindow" name="mw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mockinator V0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QComboBox" name="cbFilePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbOpenFile">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="teInputCode">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbRunMocker">
          <property name="text">
           <string>Mock Me!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabOutputCodeTab">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Header</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QTextEdit" name="teOutputHeader">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="lineWrapMode">
               <enum>QTextEdit::NoWrap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Implementation</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextEdit" name="teOutputImpl">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="lineWrapMode">
               <enum>QTextEdit::NoWrap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="sb"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSave</sender>
   <signal>clicked()</signal>
   <receiver>mw</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbRunMocker</sender>
   <signal>clicked()</signal>
   <receiver>mw</receiver>
   <slot>onMockMe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOpenFile</sender>
   <signal>clicked()</signal>
   <receiver>mw</receiver>
   <slot>onOpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSave()</slot>
  <slot>onMockMe()</slot>
  <slot>onOpenFile()</slot>
  <slot>onCBChante(string)</slot>
  <slot>onDragndrop()</slot>
 </slots>
</ui>
